pkg/store/proxy.go

```diff
@@ -44,7 +45,12 @@ type ProxyStore struct {
 	component      component.StoreAPI
 	selectorLabels labels.Labels
 
+	// responseTimeout is a short timeout for any GRPC operation
```

Comment:
IMHO it isn't necessarily short and it depends on what you call 'short'. I think it would be best to remove that adjective from here.

Reply:
ok, thanks, will do